### PR TITLE
Add config for time-interval used to check new envelopes

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1021

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -96,6 +96,7 @@ scheduling:
       ttl: PT72H # ISO-8601
       cron: 0 0 6 * * *
     check-new-envelopes:
+      time-interval: 60 # In minutes
```
Comment (rkondratowicz) [inline src/main/resources/application.yaml]: did you consider using `PT60M` instead? We use iso-8601 in link 96 and 83

Comment (aliveni) [reply src/main/resources/application.yaml]: I tried `PT60M` but when trying to get the value, `timeInterval.get(ChronoUnit.MINUTES)`  returns ` java.time.temporal.UnsupportedTemporalTypeException: Unsupported unit: Minutes`.

Comment (aliveni) [reply src/main/resources/application.yaml]: `Duration.get()` supports only a few types. I've updated to use `seconds`.

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/NewEnvelopesFinder.java [hunk 2]
```diff
@@ -49,7 +56,7 @@ public void checkNewCrimeEnvelopesCreated() {
 
     private void checkNewEnvelopesInContainers(Set<String> containers, String containersGroupName) {
         Instant toDateTime = Instant.now();
-        Instant fromDateTime = toDateTime.minus(1, HOURS); //TODO: read duration from config
+        Instant fromDateTime = toDateTime.minusSeconds(timeInterval.getSeconds());
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/NewEnvelopesFinder.java]: would
```java
Instant fromDateTime = toDateTime.minus(timeInterval);
```
work?

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/NewEnvelopesFinder.java]: Yes it works. updated now

**Pull request merged**